src/components/AreaChart.tsx

```diff
@@ -10,10 +10,11 @@ import { ICommonChartProps } from './core/base/BaseChart';
 import { convertBucketsToAFM } from '../helpers/conversion';
 import { getStackingResultSpec } from '../helpers/resultSpec';
 import { MEASURES, ATTRIBUTE, STACK } from '../constants/bucketNames';
+import { getPropsForArea, showWarningForAreaChart } from '../helpers/optionalStacking';
 
 export interface IAreaChartBucketProps {
     measures: VisualizationObject.BucketItem[];
-    viewBy?: VisualizationObject.IVisualizationAttribute;
+    viewBy?: VisualizationObject.IVisualizationAttribute | VisualizationObject.IVisualizationAttribute[];
```

Comment:
this is here due to backward compatibility ? am I right ? 

Please talk with PO for Optional Stacking :) I would expect a follow-up cleanup ticket for future major SDK version with plan how to deprecate old interface properly :)

Reply:
Yes, you're right.
I already discussed with PO, we will have a ticket to remove `stacking` prop in the near future.